src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+body {
+  margin: 20px 10px 0;
+  box-sizing: border-box;
+}
+
+.find__area {
+  left: 0;
+  right: 0;
```

Comment:
Why do you use 'left' and 'right' if this element is not positioned? (it's position is 'static')